client/rabbitmq.go

```diff
@@ -87,7 +88,42 @@ func (c *RabbitClient) handleDelivery(delivery amqp.Delivery) {
 	}
 }
 
-func (c *RabbitClient) Call(ctx context.Context, serviceName, endpoint string, req proto.Message, resp proto.Message) error {
+func (c *RabbitClient) Req(ctx context.Context, service, endpoint string, req proto.Message, resp proto.Message) error {
+
+	// Build request
+	payload, err := proto.Marshal(req)
+	if err != nil {
+		log.Errorf("[Client] Failed to marshal request: %v", err)
+		return errors.Wrap(err) // @todo custom error code
```

Comment:
Now that we've refactored this the way we like it, we could give all errors coming out of typhon a unique error code. We could say any error code < 1000 is from typhon